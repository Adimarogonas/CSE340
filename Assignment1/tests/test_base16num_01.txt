11x16
134x16
ABCGx16
