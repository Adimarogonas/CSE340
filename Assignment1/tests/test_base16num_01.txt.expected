{11x16 , BASE16NUM , 1}
{134x16 , BASE16NUM , 2}
{ABCGx16 , ID , 3}
{ , END_OF_FILE , 5}
